website/docs/r/runtime_class_v1.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "kubernetes"
+subcategory: "node/v1"
+page_title: "Kubernetes: kubernetes_runtime_class_v1"
+description: |-
+  A runtime class is used to determine which container runtime is used to run all containers in a pod. 
+---
+
+# kubernetes_runtime_class_v1
+
+A runtime class is used to determine which container runtime is used to run all containers in a pod.
+
+
+## Example usage
+
```

Comment:
Would add a simple example code block of using the `kubernetes_runtime_class_v1`